recipes-domd/domd-image-weston/domd-image-weston.bbappend

```diff
@@ -69,3 +69,10 @@ configure_versions_rcar() {
 python do_configure_append_rcar() {
     bb.build.exec_func("configure_versions_rcar", d)
 }
+
+do_install_append () {
+    local LAYERDIR=${TOPDIR}/../meta-xt-prod-gen3-test
+    find ${LAYERDIR}/doc -iname "u-boot-env*" -exec cp -f {} ${DEPLOY_DIR}/domd-image-weston/images/${MACHINE}-xt \; || true
```

Comment:
Currently it is named u-boot-env-salvator-x-h3-m3.txt. But it contains nothing SoC specific. Could you rename it to u-boot-env-salvator-x.txt for example?

Reply:
Done

Reply:
Done